pkg/addons/openebs/static/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# this file is automatically generated by buildtools. manual edits are not recommended.
+# to regenerate this file, run the following commands:
+#
+# $ make buildtools
+# $ output/bin/buildtools update addon <addon name>
+#
+version: 4.1.0
+location: oci://proxy.replicated.com/anonymous/ttl.sh/ricardox/openebs
+images:
```

Comment:
i assume these are all the images found in the chart, but not necessarily the ones we actually end up using?

Reply:
no problem. why are there so many images? the airgap builder doesn't include all of these images in the airgap bundle.

Reply:
solved as discussed.